### DISCUSSION TITLE
Android in-app browser error

### DISCUSSION THREAD

**Discussion by MONO-ON** [phantom/docs#135] [Help & Support] [answered]
When i click the mint button, and confirm in the wallet, but the code I used 'await signTransaction(transaction)' (from @solana/web3.js), it doesn't process anymore and stops working.

but, Everything is ok in iOS phantom in-app browser and web

What's the problem? my page is using React

Below is the log when debugging on Android. The part where the error occurs is after "await signTransaction(transaction)" in my code and after pressing confirm in the wallet.

In my opinion, it works fine until the user presses the confirm button in the wallet window, but there seems to be an error in the logic handled by the phantom wallet itself. (Again, I had no issues on iOS)

![image](https://github.com/phantom/docs/assets/37432366/98ff47f0-1d15-4a80-95cf-0260a14e35c3)

What is the problem???

somebody help me plz :'(

**Comment by adamdelphantom** [comment, marked as answer]
@MONO-ON Would you mind sharing a full snippet of your transaction code, please?
